test/integration/smart_answer_flows/simplified_expenses_checker_test.rb

```diff
@@ -91,25 +91,26 @@ class SimplifiedExpensesCheckerTest < ActiveSupport::TestCase
 
     context "not buying new vehicle, not claimed Capital Allowance before, expect to claim 1000 pounds, expect to drive 2000 miles, (Q2, Q3, Q4, Q8, result 2)" do
       setup do
+        add_response "used"
         add_response "no"
-        add_response "no"
-        add_response "1000" #vehicle_costs
+        add_response "1000" #price_of_vehicle
+        add_response "80" # usage
```

Comment:
The `usage` is just a label to help associated that line with the `vehicle_business_use_time` question. In hindsight, it is not very descriptive, I have replaced it with `vehicle_business_use_time`.  

The changes here are in the right place and not in previous commits